### PR TITLE
chore(web): add missing translations

### PR DESCRIPTION
**Pull request by nourbalaha**
# Overview
Add Japanese translation for meta-data and reference field

### CODE REVIEW

File: web/src/i18n/translations/ja.yml [hunk 1]
```diff
@@ -112,23 +112,23 @@ Unpublish: 非公開
...
 Updated At: 更新日時
-Updated By: ''
-Publish State: ''
-This item has been referenced: ''
-Are you going to refer to it? The previous reference will be canceled automatically: ''
-Link item: ''
+Updated By: アイテム参照
+Publish State: 公開状態
+This item has been referenced: このアイテムは参照されました
+Are you going to refer to it? The previous reference will be canceled automatically: 参照しますか？前回の参照は自動的にキャンセルされますので、よろしですか？
```
Comment (rot1024) [inline web/src/i18n/translations/ja.yml:121]:
```suggestion
Are you going to refer to it? The previous reference will be canceled automatically: 参照しますか？以前の参照は自動的に解除されますがよろしいですか？
```

File: web/src/i18n/translations/ja.yml [hunk 2]
```diff
@@ -209,40 +209,40 @@ CLOSED: クローズ済
...
+Relation: 関係
+Meta Data: メタデータ
 Add Field: フィールドを追加
 Display name: 表示名
 Please input the display name of field!: このフィールドの表示名を入力してください
 Field key must be unique and at least 1 character long. It can only contain letters, numbers, underscores and dashes.: フィールドのキーは1文字以上かつ一意である必要があります。アンダースコア、ダッシュ、文字、数値が利用できます。
 Key is not valid: 無効なキーです
 Set Options: オプションを設定
-Set Tags: ''
+Set Tags: タグをセットする
```
Comment (rot1024) [inline web/src/i18n/translations/ja.yml:223]:
```suggestion
Set Tags: タグを設定する
```

File: web/src/i18n/translations/ja.yml [hunk 3]
```diff
@@ -209,40 +209,40 @@ CLOSED: クローズ済
...
+Field: フィールド
+Previous: 前
+Next: 次
+Confirm: 確認
+Reference setting: 参照設定
+Corresponding field: 対応フィールド
+Select the model to reference: 参照するモデルを選択してください
+Please select the model!: モデルを選択してください！
+Reference direction: 参照方向
+One-way reference: 一方向参照
```
Comment (rot1024) [inline web/src/i18n/translations/ja.yml:242]:
```suggestion
One-way reference: 一方向の参照
```

File: web/src/i18n/translations/ja.yml [hunk 4]
```diff
@@ -209,40 +209,40 @@ CLOSED: クローズ済
...
+A unidirectional relationship where an item refers to another item: 項目が別の項目を参照する一方向の関係
+Two-way reference: 双方向参照
```
Comment (rot1024) [inline web/src/i18n/translations/ja.yml:244]:
```suggestion
Two-way reference: 双方向の参照
```

File: web/src/i18n/translations/ja.yml [hunk 5]
```diff
@@ -263,16 +263,16 @@ Multiple select: 複数選択
...
-Check Box: ''
-Select from a list of checkboxes: ''
-Fields: ''
+Reference: 参考文献
+Reference field: 参照フィールド
+Date: 日付
+Date picker: 日付ピッカー
+Select from a list of tags: タグのリストから選択する
+Check Box: チェックボックス
+Select from a list of checkboxes: チェックボックスのリストから選びなさい
```
Comment (rot1024) [inline web/src/i18n/translations/ja.yml:272]:
```suggestion
Select from a list of checkboxes: チェックボックスのリストから選択する
```

File: web/src/i18n/translations/ja.yml [hunk 6]
```diff
@@ -263,16 +263,16 @@ Multiple select: 複数選択
...
+Fields: フィールド
 Are you sure you want to delete this field?: このフィールドを削除します、よろしいですか？
-Publish Status: ''
+Publish Status: 発行ステータス
```
Comment (rot1024) [inline web/src/i18n/translations/ja.yml:275]:
```suggestion
Publish Status: 公開ステータス
```

File: web/src/i18n/translations/ja.yml (hunk 1 above)
Comment (nourbalaha) [reply web/src/i18n/translations/ja.yml:121]: Done!

File: web/src/i18n/translations/ja.yml (hunk 2 above)
Comment (nourbalaha) [reply web/src/i18n/translations/ja.yml:223]: Done!

File: web/src/i18n/translations/ja.yml (hunk 3 above)
Comment (nourbalaha) [reply web/src/i18n/translations/ja.yml:242]: Done!

File: web/src/i18n/translations/ja.yml (hunk 4 above)
Comment (nourbalaha) [reply web/src/i18n/translations/ja.yml:244]: Done!

File: web/src/i18n/translations/ja.yml (hunk 5 above)
Comment (nourbalaha) [reply web/src/i18n/translations/ja.yml:272]: Done!

File: web/src/i18n/translations/ja.yml (hunk 6 above)
Comment (nourbalaha) [reply web/src/i18n/translations/ja.yml:275]: Done!

**Pull request merged**